.github/workflows/release.yml

```diff
@@ -138,7 +138,7 @@ jobs:
                 notes += line
 
           with open(os.environ["GITHUB_OUTPUT"], "a", encoding="utf-8") as fh:
-              fh.writelines([f"name={name}", f"version={version}"])
+              fh.writelines([f"name={name}\n", f"version={version}\n"])
```

Comment:
@michalmo Found the issue here. I expected newlines to be included I guess, but it's not so the variables ended up on the same line.